File: android/aq-android-ben10/src/app/src/main/java/com/themeengine/activities/UserProfileActivity.java
package com.themeengine.activities;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.widget.TextView;
import com.themeengine.R;

public class UserProfileActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_user_profile);
        
        setupProfile();
    }

    private void setupProfile() {
        TextView usernameText = findViewById(R.id.username_text);
        TextView emailText = findViewById(R.id.email_text);
        // Profile management implementation would go here
    }
}

File: android/aq-android-ben18/src/app/src/main/java/co/ostorlab/myapplication/UserProfileActivity.kt
package co.ostorlab.myapplication

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class UserProfileActivity : AppCompatActivity() {
    
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_user_profile)
        
        setupProfileInfo()
    }
    
    private fun setupProfileInfo() {
        findViewById<TextView>(R.id.tvUsername).text = "John Doe"
        findViewById<TextView>(R.id.tvEmail).text = "john.doe@example.com"
        findViewById<TextView>(R.id.tvMemberSince).text = "Member since: January 2023"
        findViewById<TextView>(R.id.tvReadingStats).text = "Articles read this month: 45"
    }
}
